localization/english/CWP/spain/francoist_reform_je_l_english.yml

```diff
@@ -0,0 +1,11 @@
+﻿l_english:
+francoist_reform: "Francoist Reform"
+francoist_reform_reason: "After the defeat of fascism in the war we need to change some fundamental things to get international recognition. It is time to slowly enforce some minimal reforms, we are not fascists, aren't we...?"
+francoist_reform_goal: "Our ideology was just based on Catholicism afterall. We will use Movimiento Nacional instead of the Party"
+
+democracia_organica.t: "Demoracia Orgánica"
+democracia_organica.d: "The organic conception of society dates back to the Stoics, while its foremost theorist was Althusius. In the Contemporary Age, German idealism, with figures like Hegel and lesser-known thinkers like Karl Christian Friedrich Krause, embraced this idea. Krause's disciple, Heinrich Ahrens, systematically developed it and became one of the most dedicated proponents of Krausist philosophy. In contrast to liberal democracies, organic democracy considered that instead of the criticized political parties, the natural organs of association were three: the family, where one is born; the municipality, where one lives; the trade union, where one works."
+democracia_organica.a: "One Nation One Party"
+democracia_organica.b: "We must turn to FET y de las JONS (Falange Española Tradicionalista y de las Juntas de Ofensiva Nacional Sindicalista) as our state-party."
+democracia_organica.c: "We are a democracy in our way."
+democracia_organica.e: "We will use the Democracia Orgánica to legitimize ourselves as an alternative to liberal democracy. A step for guaranteeing that we distance ourselves from fascism."
```

Comment:
```suggestion
democracia_organica.t: "Demoracia Orgánica"
democracia_organica.d: "Our nation finds itself at a political crossroads. In which direction should we progress ourselves on the international stage?"
democracia_organica.f: "The organic conception of society dates back to the stoics. While its foremost theorist was Althusius, in the Contemporary Age figureheads such as Hegel and lesser-known thinkers like Karl Christian Friedrich Krause, embraced this political thought. Krause's disciple, Heinrich Ahrens systematically developed it and became one of the most dedicated proponents of Krausist philosophy. In contrast to liberal democracies, organic democracy considered that instead of oft criticized political parties, the natural organs of association were three: the family, where one is born; the municipality, where one lives; and the trade union, where one works."
democracia_organica.a: "One nation, one party."
democracia_organica.b: "Turn to #italic La Falange#! as our guide."
democracia_organica.c: "We are a democracy!"
democracia_organica.e: "Use #italic Democracia Orgánica#! to distance ourselves from fascism."
```

- FET y de las JONS was colloquially reffered to as "La Falange" in spain and is the name of the registered party in spain today. Pointedly I also removed "as our state-party" since franco VEHEMENTLY insisted FET y de las JONS was a "movement", not a party. 

- Moved flowery description to the flavor section and changed the description to be more direct with what we want the player to decide.